modules/gapi/include/opencv2/gapi/garray.hpp

```diff
@@ -328,17 +329,21 @@ namespace detail
 template<typename T> class GArray
 {
 public:
+    // Host type (or Flat type) - the type this GArray is actually
```

Comment:
Why are these changes here? Don't they suppose to be already in master?

Reply:
We need HT in the public constructor. Put private zone above public zone - bad behavior.

Reply:
I guess it's fine to have HT in public since it's almost the original T which is known